src/screens/UserPortal/Organizations/Organizations.tsx

```
@@ -225,94 +220,80 @@ export default function organizations(): JSX.Element {
-    } else if (mode === 2) {
+    }
+  }, [joinedOrganizationsData, mode, userId]);
+
+  // Update organizations list when createdOrganizationsData changes (mode 2)
+  useEffect(() => {
+    if (mode === 2) {
       if (
-        createdOrganizationsData?.users?.[0]?.appUserProfile
-          ?.createdOrganizations
+        createdOrganizationsData &&
+        createdOrganizationsData.user?.createdOrganizations
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)